src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsWrite.cs

```diff
@@ -13,6 +13,7 @@
 
 namespace BasicEventSourceTests
 {
+#pragma warning disable SA1136 // Enum values should be on separate lines
```

Comment:
Or you could enable the rule for product only not tests. We do that with some rules.